src/pf-notification-drawer/pf-notification-drawer.scss

```diff
@@ -0,0 +1,46 @@
+.drawer-pf {
```

Comment:
My point is there should be no CSS derivatives for patternfly-react, patternfly-ng, etc. The CSS and HTML layout should be the same for any implementation. 

The idea is that we should be able to create an example of this in the Patternfly Core test pages. Therefore, the CSS really belongs there so consumers of Patternfly Core can use it -- as well as patternfly-react, patternfly-ng, etc.

Ideally, there should be no CSS in this repo at all. Similar to Amgular Patternfly, there may be cases where we need a small amount of CSS, but it was added with the intention that it's back ported to Patternfly.